test_files/basic.untyped.js

```diff
@@ -0,0 +1,17 @@
+/** @return { ?} */ // This test is just a random collection of typed code, to
+// ensure the output is all with {?} annotations.
+function func(/** ? */ arg1) {
```

Comment:
This is actually valid syntax for inline declarations. You could write `/** @param {string} foo */` before, or `function(/** string */ foo) { ... }*/ inline. Also works for variables, `var /** string */ foo;`. It's not much nicer than regular JSDoc, but it's easier to emit because it's at the same syntactical level as the symbol itself.